tests/resources/Harbor-Pages/HomePage.robot

```diff
@@ -18,24 +18,24 @@ Resource  ../../resources/Util.robot
 
 *** Variables ***
 ${HARBOR_VERSION}  v1.1.1
-
+${timeout}  60
+${login_btn}  css=.btn
+${harbor_span_title}  xpath://span[contains(., 'Harbor')]
+${login_name}  id=login_username
+${login_pwd}  id=login_password
 *** Keywords ***
 Sign In Harbor
     [Arguments]  ${url}  ${user}  ${pw}
     Go To    ${url}
-    Sleep  5
-    ${title}=  Get Title
-    Log To Console  ${title}
-    Run Keyword If  "${title}" != "Harbor"  Capture Screenshot And Source  ELSE  Log  The title is Harbor
-    Should Be Equal  ${title}  Harbor
-    Sleep  2
+    Wait Until Element Is Visible  ${harbor_span_title}  timeout=${timeout}
```

Comment:
it could have a way to set a global timeout in the robot framework. I have already did this in https://github.com/goharbor/harbor/blob/master/tests/resources/Util.robot#L23, please double confirm whether it works?